src/UpdateLocalizedResources.targets

```diff
@@ -0,0 +1,108 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project DefaultTargets="UpdateXlf" ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="SaveXlfToResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="UpdateXlfFromResx"/>
+  <UsingTask AssemblyFile="$(BuildBinDir)\Microsoft.Build.LocalizationTasks.dll" Condition="$(LocalizedBuild) == 'true'" TaskName="EmitLocalizedResources"/>
+
+  <!-- Emit the localized EmbeddedResource items before the common targets start using them -->
+  <PropertyGroup>
+    <PrepareResourceNamesDependsOn>
+      EmitLocalizedResources;
+      UpdateLocalizedResxForEnglishResx;
+      $(PrepareResourceNamesDependsOn)
+    </PrepareResourceNamesDependsOn>
+  </PropertyGroup>
+
+  <Target Name="EmitLocalizedResources">
```

Comment:
> some parts needed to output items indifferent of incremental check

I'm not sure I understand. Can you elaborate?

> test satellites

That's definitely interesting. Maybe `Touch` an additional output if there you're doing test satellites? That would require a clean build to switch back, still.